### PR TITLE
Fix seeking on DAZN & AppleTV+ apps

### PR DESCRIPTION
**Pull request by varumugam123**
Proposed work around for - https://github.com/WebPlatformForEmbedded/WPEWebKit/issues/867

### CODE REVIEW

File: Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp [hunk 1]
```diff
@@ -309,6 +308,8 @@ bool MediaPlayerPrivateGStreamerMSE::doSeek(const MediaTime& position, float rat
                 loadingFailed(MediaPlayer::NetworkState::Empty);
             else
                 m_isSeeking = true;
+        } else if(state < GST_STATE_PAUSED && !prevSeekPending) {
```
Comment (emutavchi) [inline Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp:310]: app may trigger seek multiple times, and there is no guarantee that the app will provide data for the pending one (even though in the practice they do).  this is probably a different issue.

Comment (varumugam123) [reply Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp:310]: Will it make sense to force it on `if(state < GST_STATE_PAUSED && (!prevSeekPending || m_mediaSource->isSeeking()))`

Comment (varumugam123) [reply Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp:310]: The isSeeking() is not present right now in the MediaSourcePrivateClient...

Comment (emutavchi) [reply Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp:310]: I suspect that it wouldn't reach doSeek if seek is pending, see MediaPlayerPrivateGStreamerMSE::seek
```    
  if (m_isSeeking && m_isSeekPending) {
        m_seekTime = time;
        return;
    }
```

**Comment by calvaris** [comment]
I was told to not focus on this anymore and since I don't have a way of testing this. I am closing it.

**Pull request closed without merging**